PicoVR Unity SDK/chapter_ten.txt

```diff
@@ -0,0 +1,157 @@
+
+**Note**: Developers should release the resources dynamically loaded by the game or app.
+
+How do you apply the sensor to other objects in the scene?
+----------------------------------------------------------------------------
+
+1.  Disable the **Pvr_Unity SDK Head Track** script in the scene. 
+
+.. image:: _static/9.2.png
+
+Figure 10.4 Selecting the Pvr_Unity SDK Head Track Script.
+
+2.  Choose the objects you want to apply the sensor to, and add the **Pvr_UnitySDKHeadTrack** script.
+
+What are the framerate and scene requirements in order to achieve smooth effects with the Pico all-in-one?
+--------------------------------------------------------------------------------------------------------------------------------------------------------
+
+  **Framerate Requirements and Recommendations:**
+
+  The framerate for a Pico all-in-one scene cannot be below 30 FPS. A frame rate of 45 FPS and above is recommended.
+
+  **Scene Requirements and Recommendations:**
+
+    - Limit the number of triangle slices of the model to a maximum of 100,000 and the number of vertices of the model to a maximum of 100,000.
+    - Limit the number of regular Unity lights such as electric light sources to a maximum of 50 and the number of regular Unity particle systems to a maximum of 50.
+    - Try to use a more efficient Unity shader. 
+    - Reduce the number of model slice anchor points while ensuring correct display.
+    - Try to share materials and animations.
+    - Try to use compressed texture formats.
+    - Use as few lighting effects as possible.
+    - Use static batch processing as much as possible.
+    - Optimize scripts to reduce unnecessary consumption of memory, computing, and other resources.
+
+How do you replace the loading animation on application startup?
+----------------------------------------------------------------------------
+
+Open the Android Manifest and locate the ``plat_logo`` tag. The default value for ``android: value`` is ``0``, which means that the default loading interface is used:
+
+.. code-block:: xml
+
+  <!—Whether the platform animation is diplayed -->
+  <meta-data android:name="platform_logo" android:value="0">
+
+If you want to use the Pico proprietary loading interface, change the value of ``android:value`` to ``1``.
+
+How do you turn off the auto-connection to 3DoF when running games on mobile phones?
+-------------------------------------------------------------------------------------------
+
+On regular mobile phones, after running the Pvr_Controller Demo, the game will automatically search for controllers nearby, connect to the first Pico 3DoF controller it finds, and stop searching when connected. If it fails to connect, a popup a window will appear after 15 seconds to prompt the user to search again or stop the search.
+
+To turn off the auto-connect feature, set ``isAutoConnect`` to false in ``Pvr_ControllerLink.cs``.
+
+Does the device support the Unity Linear color space?
```

Comment:
Remove this question because Linear Color Space is supported since 2.7.9
Starting here...